misc/config_tools/data/tgl-vecow-spc-7100-Corei7/tgl-vecow-spc-7100-Corei7.xml

```diff
@@ -0,0 +1,5092 @@
+<acrn-config board="tgl-vecow-spc-7100-Corei7">
+  <BIOS_INFO>
+	BIOS Information
+	Vendor: American Megatrends International, LLC.
+	Version: E5000X2U3FIC105-BPCIe
+	Release Date: 01/18/2022
+	BIOS Revision: 5.19
+	</BIOS_INFO>
+  <BASE_BOARD_INFO>
+	Base Board Information
+	Manufacturer: Vecow Co., Ltd
+	Product Name: SPC-7100-1185GRE32
+	Version: EMBC-5000 MB Rev.B
+	</BASE_BOARD_INFO>
+  <PCI_DEVICE>
```

Comment:
Nitpick: can you run `sudo update-pciids` before running the board inspector? Not all PCI device IDs are in the latest update that's available but the majority are.

Reply:
@jackwhich , please help to run and generate board.xml, and also update board inspector document for this pre-operation.

Reply:
FIX